### PR TITLE
CNV-23499: 4.10+ HPP CSI only (4.13 RN repeat)

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.13

Issue: [CNV-23499](https://issues.redhat.com/browse/CNV-23499)

Link to docs preview: [Removed features](https://57514--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-13-release-notes.html#virt-4-13-removed) (second bullet) 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by aspauldi** [comment]
Hi kgoldbla@redhat.com, can you please take a look at this release note for 4.13?  Thanks.

File: virt/virt-4-13-release-notes.adoc [hunk 1]
```diff
@@ -117,6 +114,9 @@ Removed features are not supported in the current release.
 //CNV-19043 Release note: CHANGE Remove rhel6 support
 * Red Hat Enterprise Linux 6 is no longer supported on {VirtProductName}.
 
+//CNV-23499: Carry over/repeat removed feature from version 4.12 
+* Support for the legacy HPP custom resource, and the associated storage class, has been removed. Beginning in {VirtProductName} 4.12, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage.
```
Comment (kgoldbla) [inline virt/virt-4-13-release-notes.adoc:118]: Beginning in {VirtProductName} 4.12 - > Not sure whether you are stating when this change was made. If so it was made in OpenShift Virtualization 4.10. If you are referring to how things works in the current version (4.13) then change 'Beginning in' to 'In'

Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc]: You're absolutely right, Kevin! Thanks for your comment; I've made the change. Can you take another look? Thanks!

**Comment by aspauldi** [comment]
@kgoldbla, thanks for your review. Please take a look and let me know it all is well. Thanks, Audrey

**Review by kgoldbla — COMMENTED** [review]
lgtm

File: virt/virt-4-13-release-notes.adoc [hunk 2]
```diff
@@ -117,6 +114,9 @@ Removed features are not supported in the current release.
 //CNV-19043 Release note: CHANGE Remove rhel6 support
 * Red Hat Enterprise Linux 6 is no longer supported on {VirtProductName}.
 
+//CNV-23499: Carry over/repeat removed feature from version 4.12 
+* Support for the legacy HPP custom resource, and the associated storage class, has been removed for all new deployments. In {VirtProductName} 4.13, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. Only in the case of an upgrade will a previously installed legacy HPP custom resource still be supported.
```
Comment (eohartman) [inline virt/virt-4-13-release-notes.adoc:118]: I feel like the "Only in the case of..." sentence is a bit wordy and awkward.
Maybe change it to "Legacy HPP custom resource is supported only if it had been installed on a previous version of {VirtProductName}."

**Review by eohartman — CHANGES_REQUESTED** [review]

File: virt/virt-4-13-release-notes.adoc (hunk 2 above)
Comment (kgoldbla) [reply virt/virt-4-13-release-notes.adoc:118]: Igtm
